packages/components/src/ActionList/ActionList.tsx

```diff
@@ -75,6 +77,12 @@ export interface ActionListColumn {
   sortDirection?: 'asc' | 'desc'
 }
 
+export interface FiltersConfig {
+  availableFilters: AvailableFilter[]
+  columnsCustomizable?: boolean
+  fieldFilters?: string[]
+}
+
```

Comment:
Extend `InputFilterProps` so the interfaces are bound together.

Also `columnsCustomizeable` doesn't really require Filters so we should separate that out (see below)

```suggestion
export interface FiltersConfig extends Omit<InputFiltersConfig, 'hideFilterIcon', 'className'> {}
```